jobs/datadog-cc-worker/templates/process.yaml.erb

```diff
@@ -0,0 +1,6 @@
+init_config:
+
+instances:
+  - name: cc api worker
```

Comment:
This is more complicated to configure than what I though. I think it is enough how it is now, the wins do not justify the additional complexity